lib/configuration.yaml

```diff
@@ -14773,8 +14773,8 @@ mainnet_full: &mainnet_full
     <<: *mainnet_base_core
     genesis:
       src:
-        file: mainnet-genesis.json
-        hash: 5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb
+        - mainnet-genesis.json
+        - 5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb
```

Comment:
I suspect that is wrong, or rather that there should not be any change to that file. Commit b258f926580387f1 changed the way the config was parsed and has since been reverted.

You can probably drop these changes to this file and still have the test pass (as long as you have rebased).

Reply:
Great, let me try!